METHOD USED FOR CURRENT RUN unc_resnet50_run1_tl_2_td_256_id_448_md_56_sl_20_run1
cuda being used with 1 GPUs!!
No. of params in text_encoder: 1667072
No. of params in conv_3x3: 16519680
No. of params in sfm: 2632192
No. of params in hcam: 21239808
No. of params in aspp_decoder: 8919040
No. of params in mask_decoder: 8263425
Total number of params: 59241217
wandb: WARNING Calling wandb.run.save without any arguments is deprecated.Changes to attributes are automatically persisted.
Using cache found in /home3/theanhtran/.cache/torch/hub/pytorch_vision_v0.6.1
Initializing dataset
loading dataset refcoco into memory...
creating index...
index created.
DONE (t=8.09s)
Traceback (most recent call last):
  File "/home3/theanhtran/02_subject/Segmentation/SHNET/main.py", line 358, in <module>
    main(args)
  File "/home3/theanhtran/02_subject/Segmentation/SHNET/main.py", line 235, in main
    train_dataset = ReferDataset(
  File "/home3/theanhtran/02_subject/Segmentation/SHNET/dataloader/referit_loader.py", line 148, in __init__
    self.images += torch.load(imgset_path)
  File "/home3/theanhtran/env/anaconda/envs/3.9Segmentation/lib/python3.9/site-packages/torch/serialization.py", line 594, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home3/theanhtran/env/anaconda/envs/3.9Segmentation/lib/python3.9/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home3/theanhtran/env/anaconda/envs/3.9Segmentation/lib/python3.9/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './data/data/unc/unc_train.pth'